addons/hr/views/hr_views.xml

```diff
@@ -73,55 +73,50 @@
             groups="group_hr_user"
             sequence="95"/>
 
+        <menuitem
+                id="menu_hr_department_kanban"
+                action="hr_department_kanban_action"
+                parent="menu_hr_root"
+                groups="base.group_user"/>
+
         <menuitem
             id="menu_human_resources_configuration"
             name="Configuration"
             parent="menu_hr_root"
-            groups="hr.group_hr_user"
+            groups="group_hr_manager"
             sequence="100"/>
 
             <menuitem
                 id="menu_config_employee"
                 name="Employee"
                 parent="menu_human_resources_configuration"
-                groups="group_hr_user"
                 sequence="10"/>
 
-            <menuitem
-                id="menu_hr_department_kanban"
-                action="hr_department_kanban_action"
-                parent="menu_hr_root"
-                groups="base.group_user"/>
```

Comment:
sorry I did not see it before but in stable, it would be better to archive that menuitem instead of removing it. 🙂 

you can set `active="0"` instead of removing it. 🙂